### DISCUSSION TITLE
Restore process on new phone

### DISCUSSION THREAD

**Discussion by paulhargreaves** [TortugaPower/BookPlayer#636] [Q&A]
It's new iPhone day, so I backed up my old phone, and restored onto my new phone.
Looking in my library my books show up, but won't play.

What is the best process to get the books across, so I don't lose state?

I've tried airdropping from the old phone to the new phone that process seems broken e.g. one book is made up of lots of files so I can't share), and another is >338MB and airdrop seems to silently fail, where it asks which app to open the sent file with but it doesn't show up in Storage Management.

**Comment by GianniCarlo** [comment]
Hey @paulhargreaves ! for release 4.1.0 or 4.1.1 we disabled backing up all the files into icloud backups, due to users complaining that they already have their books in their cloud service of choice. Hmmmm I think this calls for setting up a new flag in settings for the preference of backing up the entire library.

Regarding sharing between phones, in the upcoming version we have revamped the whole app, and you'll be able to share folders/playlists not only books. The app should be able to handle linking the incoming files to existing books if there are any though when reimporting.

Right now there's no perfect solution, but I'll add the preference flag to include the files in iCloud backups, not sure if it should be off or on by default though 🤔 (thoughts @pichfl ?)

**Comment by paulhargreaves** [comment]
Thanks for considering. 

I just backup to my laptop so no iCloud. 

Maybe ask the user the question on their first import? Point them at a web page that shows the pros/cons of the selection (eg. You have 10GB of books, you're going to have a 10GB backup   vs. No backup - on restore will need to delete all books and start again)?
